minet/cli/buzzsumo/domain.py

```diff
@@ -4,17 +4,17 @@
 #
 # Logic of the `bz domain` action.
 #
-from minet.cli.utils import with_enricher_and_loading_bar
-from minet.cli.buzzsumo.utils import with_buzzsumo_fatal_errors
 from minet.buzzsumo import BuzzSumoAPIClient
-from minet.buzzsumo.constants import ARTICLES_CSV_HEADERS
+from minet.buzzsumo.types import BuzzsumoArticle
+from minet.cli.buzzsumo.utils import with_buzzsumo_fatal_errors
+from minet.cli.utils import with_enricher_and_loading_bar
 
 
 @with_buzzsumo_fatal_errors
 @with_enricher_and_loading_bar(
     title="Retrieving articles",
     unit="articles",
-    headers=ARTICLES_CSV_HEADERS,
+    headers=BuzzsumoArticle.fieldnames(),
```

Comment:
You can even just give `BuzzsumoArticle`.

Reply:
Ok.